resources/views/pages/show.blade.php

```diff
@@ -46,13 +46,13 @@
     </div>
 
 
-    <div class="container" id="page-show" ng-non-bindable>
+    <div class="container" id="page-show">
         <div class="row">
             <div class="col-md-9 print-full-width">
-                <div class="page-content">
+                <div class="page-content" ng-non-bindable>
```

Comment:
Comments have to be put inside the `page-show` container, hence moved the ng-non-bindable to a stricter scope.